README.md

```diff
@@ -16,6 +16,9 @@ Note: requires MongoDB and RabbitMQ to be running to start correctly.
 
     sudo node index.js
 
+ * http://127.0.0.1:80/ui -- MonoRail Web UI
```

Comment:
When you mention **default** port, does it mean the one hard coded in `http-service.js` or the one in config file `monorail.json`. It will be 80 if no `httpBindPort` key found in config, but we always put 8080 in config.